js/react/components/shared/Modal.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+
+function Modal({ modalTitle, modalBody, modalFooter, rightSide }) {
```

Comment:
since this is the modal you can name the props without the modal prefix

```suggestion
function Modal({ title, body, footer, sideContent }) {
```

if it's named sideContent we can extend the functionality in the future and place it to either right or left side